pkgs/applications/audio/cmt/default.nix

```
@@ -0,0 +1,35 @@
+{ stdenv
+, fetchurl
+, ladspaH
+}:
+
+stdenv.mkDerivation rec {
+  name = "cmt";
+  version = "1.17";
+
+  src = fetchurl {
+    url = "http://www.ladspa.org/download/${name}_${version}.tgz";
+    sha256 = "07xd0xmwpa0j12813jpf87fr9hwzihii5l35mp8ady7xxfmxfmpb";
+  };
+
+  buildPhase = ''
+    cd src
+    make targets
+  '';
+
+  installPhase = ''
+    substituteInPlace Makefile --replace /usr/lib $out/lib
+    mkdir -p $out/lib/ladspa
+    make install
+  '';
+
+  buildInputs = [ ladspaH ];
```

generally inputs are listed before phases, and you should prefer to use the pre/post hooks over writing new phases:

```suggestion
  buildInputs = [ ladspaH ];

  preBuild = ''
    cd src
  '';

  installFlags = [ "INSTALL_PLUGINS_DIR=${placeholder "out"}/lib/ladspa" ];
  preInstall = ''
    mkdir -p $out/lib/ladspa
  '';
```